package.json

```diff
@@ -39,26 +43,31 @@
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "4.0.0-alpha.58",
     "react": "^16.13.1",
-    "react-dom": "16.13.1"
+    "react-dom": "16.13.1",
+    "react-scripts": "4.0.3"
   },
   "devDependencies": {
+    "@babel/preset-env": "7.14.9",
+    "@babel/preset-react": "7.14.5",
+    "@babel/preset-typescript": "7.14.5",
     "@commitlint/cli": "11.0.0",
     "@commitlint/config-conventional": "11.0.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "@types/jest": "^25.1.4",
+    "@types/jest": "26.0.24",
     "@types/node": "^12.12.38",
     "@types/react": "^16.9.27",
     "@types/react-dom": "^16.9.7",
     "@typescript-eslint/eslint-plugin": "^2.26.0",
     "@typescript-eslint/parser": "^2.26.0",
     "babel-eslint": "^10.0.3",
+    "babel-jest": "^24.9.0",
     "cross-env": "^7.0.2",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.5",
     "enzyme-to-json": "3.6.1",
-    "eslint": "^6.8.0",
+    "eslint": "^6.6.0",
```

Comment:
Can we pin this and all the other "unpinned" libraries?

Reply:
No, it depends on `react-scripts`. I'll see for the other packages. Perhaps it will rather be in a update dep branch.

Reply:
OK